pipeline/Measurer.py

```diff
@@ -0,0 +1,88 @@
+
+import sqlalchemy as sa
+
+from pipeline.parameters import Parameters
+from pipeline.data_store import DataStore
+
+from models.base import SmartSession
+from models.cutouts import Cutouts
+from models.measurements import Measurements
+
+
+class ParsMeasurer(Parameters):
+    def __init__(self, **kwargs):
+        super().__init__()
+        self.photometry_method = self.add_par(
+            'photometry_method',
+            'aperture',
+            str,
+            'Type of photometry used. Possible values are "psf" and "aperture". '
+        )
+        self.aperture_radius = self.add_par(
+            'aperture_radius',
+            3.0,
+            [float, list],
+            'Radius of the aperture in pixels. Can give a list of values. '
```

Comment:
We don't want aperture radius in pixels, we want it in units of FWHM.  Seeing varies enough from one image to the next that a constant-pixel aperture radius rarely makes sense.  (It might for the "large" aperture used to figure out aperture corrections.)

For a sky-noise dominated Gaussian-shaped PSF, the optimum aperture radius is 0.6731 FWHM.

Reply:
I only put these in as example parameters... clearly we need to decide exactly what goes where. 
I think there may be cases where a variable aperture is less stable but that should also be another choice. 
For example, you may have an "aperture_units" parameter that defaults to "fwhm" but can also take "arcsec" or "pixel".